cmd/gateway-conformance/main.go

```diff
@@ -113,6 +115,13 @@ func main() {
 						Value:       "",
 						Destination: &specs,
 					},
+					&cli.StringSliceFlag{
+						Name: "skip",
+						Usage: "Accepts a test path to skip.\nCan be used multiple times.\n" +
+							"Example: --skip \"TestTar/GET_TAR_with_format=.*/Body\" --skip \"TestGatewayBlock\" --skip \"TestTrustlessCarPathing\"\n" +
+							"It uses the same syntax as the -skip flag of go test.",
+						Destination: &skips,
+					},
```

Comment:
Parity with  `go-test -skip` is :+1:
Could we include the same for `go-test -run` in this PR? 
It will make things feature-complete, and natural for anyone familiar with go, and at least remove surprises when someone tries `-skip` and `-run` – these should work the same, and if we support passing `-skip` multiple times, we should also support that for `-run` :pray: